src/node_file.cc

```diff
@@ -2227,12 +2236,22 @@ static void Mkdtemp(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+void BindingData::MemoryInfo(MemoryTracker* tracker) const {
+  tracker->TrackField("stats_field_array", stats_field_array);
+  tracker->TrackField("stats_field_bigint_array", stats_field_bigint_array);
+  tracker->TrackField("file_handle_read_wrap_freelist",
+                      file_handle_read_wrap_freelist);
+}
+
 void Initialize(Local<Object> target,
                 Local<Value> unused,
                 Local<Context> context,
                 void* priv) {
   Environment* env = Environment::GetCurrent(context);
   Isolate* isolate = env->isolate();
+  Environment::BindingScope<BindingData> binding_scope(env);
+  BindingData* binding_data = binding_scope.data;
+  if (binding_data == nullptr) return;
```

Comment:
Could this be simplified into a bool operator on `BindingScope` such that...

```c++
  if (!binding_scope) return;
```

Reply:
@jasnell Sure, done!